apps/examples/testcase/le_tc/compression/tc_compress_read.c

```diff
@@ -22,12 +22,51 @@
 #include <tinyara/config.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <errno.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <tc_common.h>
 #include <tinyara/os_api_test_drv.h>
+#include <tinyara/compression.h>
 #include "tc_internal.h"
 
+#define MAX_INPUT_SIZE 10000
+#define MAX_OUTPUT_SIZE 10000
+#define ERROR -1
```

Comment:
Let's use <sys/types.h> instead of new definition.

Reply:
done